tests/unit/disco/containers/TestDiscoPane.js

```diff
@@ -126,14 +129,21 @@ describe(__filename, () => {
   });
 
   describe('mapStateToProps', () => {
+    it('sets platform', () => {
+      const props = helpers.mapStateToProps(
+        loadDiscoResultsIntoState([]), ownProps);
+
+      expect(props.platform).toEqual('Darwin');
+    });
```

Comment:
So yeah, this should not be done here I guess. We should not rely on `ownProps` here.

Reply:
The `params` prop should be added to the `renderProps()` function, example: https://github.com/mozilla/addons-frontend/blob/master/tests/unit/amo/components/TestCollection.js#L45-L48.